src/test/java/NumbersTest.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class NumbersTest {
+
+    @ParameterizedTest
+    @ValueSource (ints = { 0, 2, 4, 50})
+    void shouldCheckIfNumberCanBeDividedByTwo (int input) {
+        //given
+        Numbers numbers = new Numbers();
+        boolean result;
```

Comment:
to boolean result bym przeniósł do when, nie widze powodu, aby to tutaj deklarować.

Reply:
poprawione